Cargo.toml

```diff
@@ -41,6 +41,7 @@ uuid = { version = "1", features = ["v4"] }
 axum = "0.6.20"
 axum-server = { version = "0.5", features = ["tls-rustls"] }
 axum-client-ip = "0.4.1"
+tower-http = { version= "0.4.3", features = ["full"] }
```

Comment:
It would be nice not to include the `full` tower-http feature set, this will increase build times.

Reply:
Here are the different feature-sets to select from: https://github.com/tower-rs/tower-http/blob/master/tower-http/Cargo.toml#L55

Reply:
@da2ce7 You are absolutely right.
I have adjusted it. I will create a new pull-request.